### ISSUE TITLE
doesn't fold long file names, marquee effect impairs usability

### ISSUE THREAD

**Issue by TjrGithub** [TeamAmaze/AmazeFileManager#1617]
expected behavior: by default, when viewing a directory, amaze should fold or truncate long file names. There should be an option to enable folding.

actual behavior: you get shown a random portion of the filename (marquee effect). There is an option to truncate instead, but no option to fold.

* install lineageos-with-microg 15.1 as of 2019-04-10 with addon_root on a Mido
* install xposed and xprivacylua, restrict nothing important
* install amaze 3.3.2 from fdroid
* go to a directory with a really long file name

**Comment by karam72** [comment]
Yes It would be great! I thing two lines for each file would be great.
